### PR TITLE
Insert devtools section in PR description

### PR DESCRIPTION
**Pull request by harupy**
<details><summary>&#x1F6E0 DevTools &#x1F6E0</summary>

[Open in GitHub Codespaces](https://codespaces.new/harupy/mlflow/pull/9963?quickstart=1)

#### Install mlflow from this PR

```
pip install git+https://github.com/mlflow/mlflow.git@refs/pull/9963/merge
```

#### Checkout with GitHub CLI

```
gh pr checkout 9963
```

---

</details>

### Related Issues/PRs

 #xxx

### How is this PR tested?

- [x] Existing unit/integration tests
- [ ] New unit/integration tests
- [x] Manual tests

https://github.com/mlflow/mlflow/assets/17039389/925f7cf6-f6a0-48d9-98b5-8b0289637cdf

### Does this PR require documentation update?

- [ ] No. You can skip the rest of this section.
- [ ] Yes. I've updated:
  - [ ] Examples
  - [ ] API references
  - [ ] Instructions

#### Is this a user-facing change?

- [ ] No. You can skip the rest of this section.
- [ ] Yes. Give a description of this change to be included in the release notes for MLflow users.

#### What component(s), interfaces, languages, and integrations does this PR affect?

Components

- [ ] `area/artifacts`: Artifact stores and artifact logging
- [ ] `area/build`: Build and test infrastructure for MLflow
- [ ] `area/docs`: MLflow documentation pages
- [ ] `area/examples`: Example code
- [ ] `area/gateway`: AI Gateway service, Gateway client APIs, third-party Gateway integrations
- [ ] `area/model-registry`: Model Registry service, APIs, and the fluent client calls for Model Registry
- [ ] `area/models`: MLmodel format, model serialization/deserialization, flavors
- [ ] `area/recipes`: Recipes, Recipe APIs, Recipe configs, Recipe Templates
- [ ] `area/projects`: MLproject format, project running backends
- [ ] `area/scoring`: MLflow Model server, model deployment tools, Spark UDFs
- [ ] `area/server-infra`: MLflow Tracking server backend
- [ ] `area/tracking`: Tracking Service, tracking client APIs, autologging

Interface

- [ ] `area/uiux`: Front-end, user experience, plotting, JavaScript, JavaScript dev server
- [ ] `area/docker`: Docker use across MLflow's components, such as MLflow Projects and MLflow Models
- [ ] `area/sqlalchemy`: Use of SQLAlchemy in the Tracking Service or Model Registry
- [ ] `area/windows`: Windows support

Language

- [ ] `language/r`: R APIs and clients
- [ ] `language/java`: Java APIs and clients
- [ ] `language/new`: Proposals for new client languages

Integrations

- [ ] `integrations/azure`: Azure and Azure ML integrations
- [ ] `integrations/sagemaker`: SageMaker integrations
- [ ] `integrations/databricks`: Databricks integrations

<a name="release-note-category"></a>

#### How should the PR be classified in the release notes? Choose one:

- [x] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [ ] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

**Review by serena-ruan — APPROVED** [review]
LGTM! This is awesome!

File: .github/workflows/advice.js [hunk 1]
```diff
@@ -29,8 +29,30 @@ module.exports = async ({ context, github }) => {
...
+
+#### Checkout with GitHub CLI
+
+\`\`\`
+gh pr checkout ${issue_number}
+\`\`\`
+
+</p>
+</details>
+`.trimStart();
```
Comment (harupy) [inline .github/workflows/advice.js:55]: /ai are there any obvious mistakes?

File: .github/workflows/advice.js [hunk 2]
```diff
@@ -29,13 +29,35 @@ module.exports = async ({ context, github }) => {
...
+
+</p>
+</details>
+`.trim();
     await github.rest.pulls.update({
       owner,
       repo,
       pull_number: issue_number,
-      body: `${codespacesBadge}\n\n${body}`,
+      body: `${newSection}\n\n${body}`,
```
Comment (harupy) [inline .github/workflows/advice.js:60]: /ai review this change

Comment (harupy) [reply .github/workflows/advice.js:60]: /ai any obvious mistakes?

**Pull request merged**